docs/jwst/badpix_selfcal/description.rst

```diff
@@ -0,0 +1,52 @@
+Description
+===========
+
+:Class: `jwst.badpix_selfcal.BadpixSelfcalStep`
+:Alias: badpix_selfcal
+
+Overview
+--------
+The ``badpix_selfcal`` step flags bad pixels in the input data using a self-calibration 
+technique based on median filtering along the spectral axis. 
+When background exposures are available, those are used in combination with the science
+exposure to identify bad pixels; when unavailable, the step will be skipped with a warning
+unless the ``force_single`` parameter is set True. In that case, the science data alone is
+used as its own "background".
```

Comment:
Needs updating, since it might not be just background exposures. Maybe "additional exposures"?